forms_pro/api/team.py

```
@@ -77,3 +89,131 @@ def switch_team(team_id: str) -> None:
+    Toggle the can_edit_team permission for a team member
+    """
+
+    if not frappe.has_permission(
+        doctype="FP Team",
+        ptype="write",
+        doc=team_id,
+        user=frappe.session.user,
+    ):
+        raise frappe.PermissionError(
+            "You do not have permission to toggle the can_edit_team permission for this team member"
+        )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)